src/include/packager/repositories_include.rb

```diff
@@ -1,15 +1,13 @@
 # encoding: utf-8
 
-# File:	packager/repositories_include.ycp
+# File:	packager/repositories_include.rb
 #
 # Author:	Cornelius Schumacher <cschum@suse.de>
 #		Ladislav Slezak <lslezak@suse.cz>
 #		Lukas Ocilka <locilka@suse.cz>
 #
 # Purpose:	Include file to be shared by yast2-packager and yast2-add-on
 #
-# $Id$
-#
```

Comment:
in general whole this header can be deleted and only move Purpose to module as yardoc